### DISCUSSION TITLE
Icon Request: fa-astrology-planet-* (12 icons)

### DISCUSSION THREAD

**Discussion by josephmilnes** [FortAwesome/Font-Awesome#21225] [Icon Requests]
Request for Astrology Icon Set: Planets

All 9 Planets including Sun + Moon

![image](https://cloud.githubusercontent.com/assets/556268/4859198/9c4eb856-60e7-11e4-994d-41fd8c5f78d1.png)

**Comment by tagliala** [comment]
Sun and moon as "planets"? 

I suppose this one doesn't feet: http://fontawesome.io/icon/sun-o/

Could you also provide single color, 14px suitable examples of what a planet icon could be?

**Comment by josephmilnes** [comment]
I know it sounds weird. ^^

Here is an example of the planetary glyphs, astrologers just call them all planets because its easier. they are all important.

http://en.wikipedia.org/wiki/File:Planets_in_astrology_glyphs.jpg

**Comment by tagliala** [comment]
ah got it. not weird at all, thanks

**Comment by scherre** [comment]
+1

**Comment by Magidragon-Sapph** [comment]
+1

**Comment by Twanneman** [comment]
+1

**Comment by qck1** [comment]
+1 This is something I would have expected to be in a font set since they are extremely common.

**Comment by asialee6** [comment]
+1

**Comment by faospark** [comment]
I think fontawesome already has a degree of  support to planetary symbols given that male and female are the glyphs for mars and venus respectively. invert the glyph of venus will result to a variation of earth. also for now let us refer to the glyphs of planets as astronomy glyphs/symbols/icons not astrology.

There is a pretty decent unicode support for classical 9 planets but it would be AWESOME (pun intended) if fontawesome would be at the forefront in supporting glyphs for other objects that under the geophysical definition of planets are considered planets. hence forth in order of orbital radius minus earth... it would be great if fontawesome will go all in with the current IAU 8 plus Ceres, Pluto, Haumea, Makemake, and Eris
![screenshot_216](https://user-images.githubusercontent.com/11664809/40233286-f688a260-5ad4-11e8-97f1-366223cc61e4.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by asialee6** [comment]
Hi Jory,

This is great news!  I just want to mention that I'm an astrologer and
created the TimePassages Astrology App.  We've developed our own astrology
font set and I'd be happy to advise on any astrology related questions.

Excited for this,

[image: upload image]
*Asia Seltzer*
Co-owner, Chief of Operations
TimePassages App from Astrograph Software
ASTROGRAPH.COM <https://astrograph.com/>
[image: facebook] <https://www.facebook.com/timepassagesapp>
[image: instagram] <https://www.instagram.com/timepassagesapp/>
[image: appStoreBadge]
<https://apps.apple.com/us/app/timepassages/id488946918> [image:
googlePlayBadge]
<https://play.google.com/store/apps/details?id=com.astrograph.timepassages>
